pkg/transformer/kubernetes/kubernetes_test.go

```diff
@@ -499,3 +500,14 @@ func TestInitPodSpec(t *testing.T) {
 		t.Fatalf("Pod object not found")
 	}
 }
+
+func TestConfigTmpfs(t *testing.T) {
+	name := "foo"
+	k := Kubernetes{}
+	resultVolumeMount, resultVolume := k.ConfigTmpfs(name, newServiceConfig())
+
+	if resultVolumeMount[0].Name != "foo-tmpfs0" && resultVolume[0].EmptyDir.Medium != "Memory" {
```

Comment:
One last thing. Should it be `||` instead of '$$'?

What if only one of them is wrong?